src/docs/patterns/appbar.mdx

```diff
@@ -47,8 +47,6 @@ The basic app bar provides information about the current screen. Typically, this
             'https://pxblue-components.github.io/angular/?path=/story/components-dropdown-toolbar--with-full-config'
         }
         react={'https://pxblue-components.github.io/react/?path=/story/components-dropdown-toolbar--with-full-config'}
-        ionic
```

Comment:
You could leave ionic here and point it to Angular if you want

Reply:
I think it's best to avoid pointing to the same resource.